web_console/web-console.adoc

```diff
@@ -21,8 +21,8 @@ Platform 4.x Tested Integrations] page before you create the supporting
 infrastructure for your cluster.
 
 include::modules/web-console-overview.adoc[leveloffset=+1]
-include::modules/multi-cluster-about.adoc[leveloffset=+1]
-include::modules/enabling-multi-cluster-console.adoc[leveloffset=+2]
+// include::modules/multi-cluster-about.adoc[leveloffset=+1]
```

Comment:
My only comment is whether these lines in both files can just be deleted instead of just commented out?

Also, should the module files actually be removed (or moved the unused_topics directory)? I'd recommend deleting the lines/removing the modules, unless they are expected to be needed again (then it's fine as is!).

Reply:
Thanks, @bergerhoffer ! This feature is on hold for a few releases. Depending on what they decide to do with the feature, I expect to probably need this content again soon.